src/audio/music.rs

```diff
@@ -48,55 +47,52 @@ unsafe impl Send for Music<'_> {}
 unsafe impl Sync for Music<'_> {}
 
 /// Creating and opening
-impl<'src> Music<'src> {
+impl<'src, S> Music<'src, S> {
     /// Create a new (empty) `Music`.
     pub fn new() -> SfResult<Self> {
         Ok(Self {
             music: unsafe { ffi::audio::sfMusic_new() },
-            _stream: PhantomData,
+            stream: None,
         })
     }
-    /// Create a new `Music` by opening a music file
-    ///
-    /// See [`Self::open_from_file`].
-    pub fn from_file(filename: &str) -> SfResult<Self> {
```

Comment:
Why does from_file need to be removed? I'm not well versed in the music stuff, so some explanation may be needed.